pkg/controller/nodes/subworkflow/launchplan/admin.go

```diff
@@ -24,6 +24,8 @@ import (
 
 // Executor for Launchplans that executes on a remote FlyteAdmin service (if configured)
 type adminLaunchPlanExecutor struct {
+	Executor
```

Comment:
I am confused, isnt this the actual struct?

Reply:
or is admin client implementing both methods. If so why we do need to compose of the actual interfaces?

Reply:
I can delete this.